manifests/03-cluster-operator.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: config.openshift.io/v1
+kind: ClusterOperator
+metadata:
+  name: cluster-storage-operator
+  spec: {}
```

Comment:
In the linked document, this appears to be a namespaced resource?

Reply:
they aren't, seems the doc is outdated.  Also, there is a `versions` array, not a single `version` now.

Reply:
hmm this is suspect: " Also note that the ClusterOperator resource in /manifests is only a communication mechanism, to tell the ClusterVersionOperator, which ClusterOperator resource to wait for. The ClusterVersionOperator does not create the ClusterOperator resource, this and updating it is the responsibility of the respective operator."